swapnil/online_photo_system/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <title>Online Photography System</title>
+        <meta name="description" content="This contain information about Online photography system">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+    <body>
+        <div class="container">
+            <div class="nav">
+                <ul>
+                    <li><a class="active" href="index.html">Home</a></li>
+                    <li><a href="photos.html">Photos</a></li>
+                    <li><a href="bio.html">Bio</a></li>
+                    <li><a href="services.html">Services</a></li>
+                    <li><a href="contact.html">Contact</a></li>
+                </ul>
+            </div>
+            <div class="image">
+                <img src="images/1.jpg" height="100%" width="100%" alt="background">
+                <div class="text-block">
+                    <h1>Online Photography System</h1>
+                    <p style="font-size:40px">Photography Make us happy Take a shot.</p>
+                </div>
+            </div>
+            <div class="footer">
+                <p>All Rights Reserved By Swapnil Sangvikar </p>
+            </div>
+        </div>
+    </body>
+</html>
```

Comment:
add more informations and contents on home page 
add more content for footer 
use logo for naviagation bar  or name for your project 
use proper font font properties for this navigation bar and and other homepage contents also
use more css properties